scripts/migrate-vfolder-structure.py

```diff
@@ -0,0 +1,42 @@
+import sys
+from pathlib import Path
+
+
+def main():
+    input_path = Path(sys.argv[1])  # full path for vroot/local
+    output_path = Path(sys.argv[2])  # full path for volume directory. ex. vfs/
+    print(input_path, output_path)
+    subfolders = [x for x in input_path.iterdir() if x.is_dir()]
+
+    print("Subfolders ", subfolders)
```

Comment:
Should we print all of the vfolder names (they are just a bunch of UUIDs...)?
I think just showing the number of folders would be sufficient information for the user.